inc/functions_calendar.php

```diff
@@ -130,7 +130,7 @@ function build_mini_calendar($calendar, $month, $year, &$events_cache)
 
 			$link_to_day = false;
 			// Any events on this specific day?
-			if(@count($events_cache["$day-$calendar_month-$calendar_year"]) > 0)
+			if(!empty($events_cache["$day-$calendar_month-$calendar_year"]) && @count($events_cache["$day-$calendar_month-$calendar_year"]) > 0)
```

Comment:
Looks like the `count()` check is now redundant.

Reply:
Yes, it is redundant, we can remove it.